docs/source/roadmaps/may-2024.md

```diff
@@ -0,0 +1,117 @@
+# May 2024
+This roadmap is split into "themes" which roughly mirror our current priorities.
+
+## Data management
+**Current status**
+
+We have developed [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/), a simple, standard data folder 
+specification for (systems) neuroscience and [datashuttle](datashuttle.neuroinformatics.dev), to automate the 
+creation, validation and transfer of NeuroBlueprint projects. For more details on this specific project, see the 
+[datashuttle roadmap]().
+
+**Q4 2024**
```

Comment:
I'm a bit stuck with the timelining for the data-management section. For all other sections there are clear subgoals towards a specific aim (e.g. spikewrap for automating ephys preprocessing). For this it makes sense to have hard deadlines. But data-management is not really a core NIU goal and instead exists to support the other tools. If people are asking for a specific feature like metadata, or it is required for our other goals (e.g. multimodal analysis), or someone willing comes to implement it then we will do it. But in and of itself it's not a priority.

I'm stuck because on the one hand its very useful to have timelines for all roadmaps. On the other hand in this case I think any timeline is too arbitrary. And it also gives the wrong impression that these are immovable and we have dedicated hours for their implementation, when in reality we would move them up on request and / or welcome them as contributions. So for the data-management section only, I might suggest having a more general list of priorities and a section saying that these are shaped by user requirements, please get in contact if they will be useful for you or you would like to support their implementation.

Reply:
I get this point, but I disagree. We have some kind of timetable for all of our work, even if it's very (very) vague (now, next year, never). If we plan on doing it, we should decide on a timetable (even if that's based on 1% effort), and invite feedback. If we aren't going to do it, then it shouldn't be here.

I'm very wary of this roadmap just being the issue list converted into prose. It should allow researchers to plan in some way. Anything without a date doesn't achieve that goal.

These are not deadlines though of course, we can, and will, move them regularly as priorities change.

Reply:
Ok I agree, my main worry was that it will make life more difficult to researchers, because they will expect things by certain dates that don't appear, or assume things won't be delivered until much later than they could actually be. However, having this as a guide will be useful even if not accurate as it will prompt them to ask us about these goals in a way that vaguer aims wouldn't (e.g. I thought you'd have metadata by now, where is it?) (I see you don't plan to convert to NWB until 2025, can you move it up?).

Reply:
> (e.g. I thought you'd have metadata by now, where is it?) (I see you don't plan to convert to NWB until 2025, can you move it up?

That's what I'm hoping for.

I think all we can do is try our best to come up with estimates, and then try our best to meet them. If we can't for whatever reason, we communicate that. The 5th quad will be 2-3 years late, so I think people will understand!

One of the reasons why I want the whole team to review this is to make sure that the dates in each section seem appropriate. Feel free to suggest changes if the timing is wrong.